pkg/theme/theme.go

```diff
@@ -44,6 +47,11 @@ func UpdateTheme(themeConfig config.ThemeConfig) {
 	InactiveBorderColor = GetGocuiStyle(themeConfig.InactiveBorderColor)
 	SelectedLineBgColor = GetTextStyle(themeConfig.SelectedLineBgColor, true)
 	SelectedRangeBgColor = GetTextStyle(themeConfig.SelectedRangeBgColor, true)
+	
+	var cherryPickedCommitBgTextStyle =	GetTextStyle(themeConfig.CherryPickedCommitBgColor, true)
+	var cherryPickedCommitFgTextStyle =	GetTextStyle(themeConfig.CherryPickedCommitFgColor, false)
```

Comment:
small convention thing
```suggestion
	cherryPickedCommitFgTextStyle :=	GetTextStyle(themeConfig.CherryPickedCommitFgColor, false)
```